Automattic/jetpack
# Jetpack Live Branches: Adding options to populate CRM and Woo data

## Proposed changes:

* This PR adds two check-boxes to the Live Branches script options - one to populate with CRM data and the other to populate with Woo data.

### Other information:


## Jetpack product discussion

pbhBOz-2W5-p2

## Does this pull request change what data or activity we track or use?



## Testing instructions:

* Install the patched version of the script using a browser extension such as TamperMonkey: [update/live-branches-crm/tools/jetpack-live-branches/jetpack-live-branches.user.js (raw)](https://github.com/Automattic/jetpack/raw/update/live-branches-crm/tools/jetpack-live-branches/jetpack-live-branches.user.js)*
* Visit a CRM PR (ay PR in the Jetpack monorepo that touches CRM plugin files - files changed will include any file within `projects/plugins/crm`) and try selecting the 'Populate CRM data' or 'Populate Woo data' options without the CRM or Woo plugins having been selected (you should see a message saying to select the appropriate one). With the Woo or CRM plugins selected and the corresponding populate data checkboxes selected, try clicking the generated URL. It should load a JN site with populated data (with CRM this includes adding an additional plugin 'Jetpack CRM Sample Data Generator'.
* For anyone who hasn't used the Live Branches script before - it adds the link and expandable section to select options at the bottom of each PR description.
